### PR TITLE
Concepts article - long polling

### PR DESCRIPTION
**Pull request by Srushtika**

### CODE REVIEW

File: content/concepts/long-polling.textile [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+h2(#a-brief-history-how-did-long-polling-come-about). A brief history - How did long polling come about?
+
+In the early days of the web, HTTP made sense as a simple request-response protocol because it was designed as a way to distribute structured documents to interested readers, each document generally having links to others that could be browsed at the reader's leisure. A request-response model was the right strategy because a browser needed only make a request for the resource that the user wished to consume, and the job was done. The technology of the time was vastly more limited than that of today, and holding a connection open would not only have been a significant issue for web server load management but would have had little-to-no value anyway, concerning the goals and use cases that the early web was designed to solve. Early browsers such as Mosaic and the first version of Netscape Navigator were implemented with this in mind and had no support for JavaScript or other features that might position the web as anything more than a distributed resource-sharing network.
```
Comment (mattheworiordan) [inline content/concepts/long-polling.textile]: Some of my comment from the WS concepts article apply here. Also, I think it's important that we link back & forth between our articles.  In addition, we will later need to think about how we build up SEO juice for these articles by linking from other locations, but let's review that after

File: content/concepts/long-polling.textile [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+Long polling can be implemented via JSONP if absolutely necessary. That said, given the time and effort - not to mention the inefficiency of resource consumption - involved in implementing these approaches, care should be taken to assess whether their support is worth the added cost when developing new applications and system architectures. In many cases, you may be able to get away with exclusive support for something more modern, such as "WebSockets":/concepts/websockets. Alternatively, you may wish to offload the management of these sorts of concerns to a specialist cloud provider such as "Ably":https://ably.io.
```
Comment (mattheworiordan) [inline content/concepts/long-polling.textile]: We refer to Ably as "Ably Realtime" in links for SEO reasons.  Also, please use www.ably.io as opposed to ably.io

File: content/concepts/long-polling.textile [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+* Pushing data to a channel
+* Registering a client
+* Monitoring the server
+* Reading data from channels
+
+You can find out more about this library from the original "GitHub repo":https://github.com/jedisct1/Simple-Comet-Server.
+
+h2(#ably-and-long-polling). Ably and long polling
+
+At Ably we've implemented a robust suite of transports in the client libraries that we make available for communication with our servers, with WebSockets being the primary protocol for realtime communications. However, long polling is fully-supported as an automatic fallback transport when newer standards such as WebSockets are not available on the client-side. Find out more on the "Ably home page":https://ably.io.
```
Comment (mattheworiordan) [inline content/concepts/long-polling.textile]: Could users not find out more on a better page such as our platform page which talks about transports etc?

File: content/concepts/long-polling.textile [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+
+h2(#references-and-further-reading). References and further reading
+
+* "IETF: Known Issues and Best Practices for the Use of Long Polling and Streaming in Bidirectional HTTP":https://tools.ietf.org/id/draft-loreto-http-bidirectional-07.html
```
Comment (mattheworiordan) [inline content/concepts/long-polling.textile]: What about a link to our WebSocket oncept article and vice versa?

Comment (Srushtika) [reply content/concepts/long-polling.textile]: I did link the articles from within the main text, but I agree it should be in 'further reading' too. Updated.

File: content/concepts/long-polling.textile (hunk 2 above)
Comment (Srushtika) [reply content/concepts/long-polling.textile]: Sure, updated. Thanks

**Pull request merged**